src/solver/variable/include/antares/solver/variable/economy/links/flowQuad.h

```diff
@@ -268,6 +269,8 @@ class FlowQuad: public Variable::IVariable<FlowQuad<NextT>, NextT, VCardFlowQuad
 private:
     uint pLinkGlobalIndex;
     uint pNbHours;
+    std::vector<std::vector<double>>* transitMoyenInterco;
```

Comment:
About `std::vector<std::vector<double>>* transitMoyenInterco;`

Why not a reference to the vector (instead of a raw pointer) ?
It would avoid uses of symbol `&` and notation` (*transitMoyenInterco)`, always a bit harder to read.

Reply:
True, I missed this point

Reply:
Maybe adding a const then : 
`const std::vector<std::vector<double>>* transitMoyenInterco;`